src/components/dialogs/PaymentErrorDialog.vue

```diff
@@ -1,22 +1,43 @@
 <template>
   <v-dialog v-model="dialog" width="45rem" persistent :attach="attach" content-class="payment-error-dialog">
     <v-card>
-      <v-card-title>Unable to Process Payment</v-card-title>
+      <v-card-title id="dialog-title">Unable to process payment</v-card-title>
 
-      <v-card-text>
-        <p class="genErr">We are unable to process payments at this time.</p>
-        <p class="genErr">Your filing has been saved as a DRAFT and you can resume
-          your filing from the Business Dashboard at a later time.</p>
+      <v-card-text id="dialog-text">
+        <!-- display common message -->
+        <div class="genErr">
+          <p>We are unable to process your payment at this time. This {{filingName}} has been saved
+            as a DRAFT and you can retry your payment from the Business Dashboard at a later time.</p>
+        </div>
+
+        <!-- display generic message (no errors or warnings) -->
+        <div class="genErr" v-if="!isRoleStaff && (numErrors + numWarnings) < 1">
+          <p>PayBC is normally available:</p>
+          <ul>
+            <li>Monday to Friday: 6:00am to 9:00pm</li>
+            <li>Saturday: 12:00am to 7:00pm</li>
+            <li>Sunday: 12:00pm to 12:00am</li>
+          </ul>
+        </div>
+
+        <!-- display errors -->
+        <div class="genErr mb-4" v-if="numErrors > 0">
+          <p>We were unable to process your payment due to the following errors:</p>
+          <ul>
+            <li v-for="(error, index) in errors" :key="index">{{ error.error }}</li>
```

Comment:
Should this be `error.message` like the other UI's?

Same below on line 35, maybe `warning.message`?

Reply:
Ah yes. Thanks!

Reply:
I'll update the unit tests also :P

Reply:
right on!